cmd/kube-apiserver/app/options/plugins.go

```diff
@@ -83,4 +81,42 @@ func RegisterAllAdmissionPlugins(plugins *admission.Plugins) {
 	setdefault.Register(plugins)
 	resize.Register(plugins)
 	pvcprotection.Register(plugins)
+
+	// Set recommended plugins order
+	recommendedOrder := []string{
+		autoprovision.PluginName,
+		exists.PluginName,
+	}
+	recommendedOrder = append(recommendedOrder, admission.RecommendedPluginOrder...)
```

Comment:
Isn't it odd to wrap the existing recommended order like this and then override it later? It assumes things about the previous content of that var.

Reply:
For a coherent order, I think all should be specified here at once

Reply:
SGTM